telco-hub/configuration/reference-crs-kube-compare/compare_ignore

```diff
@@ -36,6 +36,9 @@ required/acm/acmPerfSearch.yaml
 # compare does not include the full policy content due to policy templating.
 required/acm/thanosSecretPolicy.yaml
 required/gitops/addPluginsPolicy.yaml
+# we make usage of Regex Inline Diff Function,
+# that does not work with local compare
+required/acm/observabilityMCO.yaml
```

Comment:
This should not be needed (I can help with the CI issues if you need), but also if we switch to the simpler template mechanism to keep the retention resolutions identical [here](https://github.com/openshift-kni/telco-reference/pull/362/files#r2437174879) this should also be unneeded.